win32/scripts/pywin32_postinstall.py

```diff
@@ -655,9 +655,13 @@ def main():
 
     * Typical usage:
 
-    > python pywin32_postinstall.py -install
+    > python -m pywin32_postinstall -install
 
-    This should be run automatically after installation,
+    * or (shorter but you don't have control over which python environment is used)
+
+    > pywin32_postinstall -install
+
+    This should be run automatically after installation when installing from source,
     but if it fails you can run it again.
```

Comment:
```suggestion
```

Just tested and the "when installed from source" part isn't even true because `self.root=build\bdist.win-amd64\wheel` when running `pip install .`. So `class my_install` right now is complete dead code. I'll tackle that in a different PR.